ansible/group_vars/all.yml

```
@@ -36,6 +36,12 @@ virtual_machines:
     project: rhel-s390x-cloud
     families:
       - rhel-8-6-s390x
+  
+#We have taken families name as "p" only there is limit on chars in instance name  in ibm cloud with 56 chars most
+  rhel-ppc64le:
```

[nit] need to be indented in line with the `rhel-ppc64le:` line, and separated across a couple of lines
```suggestion
  # We have taken families name as "p" only. There is limit on chars in instance name 
  # in ibm cloud with 56 chars max
  rhel-ppc64le:
```